views/pools/header/action-group.tsx

```diff
@@ -10,7 +10,7 @@ import { ActionGroupProps } from './header.types';
 const ActionGroup: FC<ActionGroupProps> = ({ showSearchView }) => {
   const { push } = useRouter();
 
-  const gotoFindPool = () => push(Routes[RoutesEnum.FindPool]);
+  const gotoFindPool = () => push(Routes[RoutesEnum.CreatePool]);
```

Comment:
Do not forget to keep Find Pool when u recover the Find Pool Page

Reply:
Solved in other branch/PR